src/plugins/package/azurePackagePlugin.ts

```diff
@@ -1,13 +1,14 @@
 
 import Serverless from "serverless";
-import { ServerlessCliCommand } from "../../models/serverless";
+import { ServerlessCliCommand, FunctionAppOS } from "../../models/serverless";
 import AzureProvider from "../../provider/azureProvider";
 import { PackageService } from "../../services/packageService";
 import { AzureBasePlugin } from "../azureBasePlugin";
 
 export class AzurePackagePlugin extends AzureBasePlugin {
   private bindingsCreated: boolean = false;
   public provider: AzureProvider;
+  private isLinuxTarget: boolean;
```

Comment:
This is used in multiple hooks. Removed from base plugin, but I still think it makes sense to have here.